iib/web/models.py

```diff
@@ -241,13 +241,21 @@ def get_or_create(cls, name):
         return operator
 
 
-class RequestBuildTag(db.Model):
+class BuildTag(db.Model):
     """Extra tag associated with built index image."""
 
     id = db.Column(db.Integer, primary_key=True)
     name = db.Column(db.String, nullable=False, unique=False)
+
+
+class RequestBuildTag(db.Model):
+    """Extra tag associated with built index image."""
```

Comment:
This line is the same as on line no. 245. Is it really the same?

Reply:
right, I'll fix the description